### PR TITLE
[Misc] Update dragonwell version to 8.0.0

### PR DESCRIPTION
**Pull request by yfxhust**
Summary:
Update dragonwell version to 8.0.0

Test Plan: make.sh saint check.

Reviewed-by: D-D-H, mmyxym, luchsh

Issue: https://github.com/alibaba/dragonwell8/issues/30

### CODE REVIEW

File: make.sh [hunk 1]
```diff
@@ -117,9 +121,9 @@ done
 
 # check version string
 $NEW_JAVA_HOME/bin/java -version > /tmp/version.out 2>&1
-grep "^OpenJDK Runtime" /tmp/version.out | grep "(Dragonwell $DISTRO_VERSION)"
+grep "^OpenJDK Runtime" /tmp/version.out | grep "(Alibaba Dragonwell $DISTRO_VERSION)"
```
Comment (luchsh) [inline make.sh]: pls add `-${BUILD_INDEX}` to the end of the pattern.

**Comment by luchsh** [comment]
Could you please show us what the new output of `Dragonwell8/bin/java -version` looks like now?

File: make.sh (hunk 1 above)
Comment (yfxhust) [reply make.sh]: DISTRO_VERSION already contains BUILD_INDEX.
I think it is not necessary.

**Comment by yfxhust** [comment]
```
Finished building OpenJDK for target 'images'
openjdk version "1.8.0_202"
OpenJDK Runtime Environment (Alibaba Dragonwell 8.0.0-bb66) (build 1.8.0_202-b01)
OpenJDK 64-Bit Server VM (Alibaba Dragonwell 8.0.0-bb66) (build 25.202-b01, mixed mode)
java.vm.vendor="Alibaba"
java.vendor.url=http://www.alibabagroup.com
java.vendor=Alibaba
java.vendor.url.bug=mailto:dragonwell_use@googlegroups.com
OpenJDK Runtime Environment (Alibaba Dragonwell 8.0.0-bb66) (build 1.8.0_202-b01)
OpenJDK 64-Bit Server VM (Alibaba Dragonwell 8.0.0-bb66) (build 25.202-b01, mixed mode)
        libz.so.1 => /lib64/libz.so.1 (0x00007ffbb6b15000)
```
This is the output for the jdk applied my patch.
bb66 is my magic build number.   I think it is as same as what I expect.

**Comment by yfxhust** [comment]
I think --with-build-number can work.

**Pull request merged**